tests/pom.xml

```diff
@@ -463,20 +468,28 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     <from>${docker.image.org-name}/hono-adapter-http-vertx:${project.version}</from>
                     <assembly>
                       <mode>dir</mode>
-                      <basedir>/etc/hono</basedir>
+                      <basedir>/</basedir>
                       <inline>
                         <id>config</id>
+                        <dependencySets>
+                          <dependencySet>
+                            <includes>
+                              <include>io.netty:netty-tcnative-boringssl-static</include>
```

Comment:
did you create a CQ for this test dependency?

Reply:
Irgs … no, I did not. That is a works-with anyway, as it is build & test only. But that will most likely be too late for this release.

I will try to get a CQ for that ASAP, if it doesn't work out, we can always remove that unit test and re-add it a later time.

Reply:
ok